doc/model_details/noise_generator.ipynb

```diff
@@ -309,14 +309,14 @@
     "    ng = nest.Create('noise_generator', params={'mean': mu, 'std': sigma, 'dt': dt})\n",
     "    vm = nest.Create('voltmeter', params={'interval': resolution})\n",
     "    nrns = nest.Create('iaf_psc_alpha', N, params={'E_L': 0., 'V_m': 0., 'V_th': 1e6,\n",
-    "                                                'tau_m': tau_m, 'C_m': C_m})\n",
+    "                                                   'tau_m': tau_m, 'C_m': C_m})\n",
     "    nest.Connect(ng, nrns, syn_spec={'delay': delay})\n",
     "    nest.Connect(vm, nrns)\n",
     "    \n",
     "    nest.Simulate(t_max)\n",
     "    \n",
     "    # convert data into time axis vector and matrix with one column per neuron\n",
-    "    ev = nest.GetStatus(vm, keys=['events'])[0][0]\n",
+    "    ev = vm.events\n",
```

Comment:
Can we remove this and rather use `vm.events` directly instead of `ev` in the next line?